plugins/cisco_umbrella_destinations/plugin.spec.yaml

```diff
@@ -0,0 +1,534 @@
+      required: false
+  destinations:
+    destination:
+      type: string
+      description: Destination can be domain, url or ip | e.g. <domain_name>
+      required: true
+    type:
+      type: string
+      description: Type can be DOMAIN, URL, IPV4 | e.g. DOMAIN
+      required: true
+    comment:
+      type: string
+      description: e.g. Added new destination list
+      required: false
+  dResponse:
+    id:
+      title: ID
+      description: Unique id of the destination list.
+      type: integer
+      required: false
+    organizationId:
+      title: Organization Id
+      description: ID of org, e.g. 2345678
+      type: integer
+      required: false
+    access:
+      title: Access
+      description: Access can be allow or block. It defines destinationList type, e.g. allow
+      type: string
+      required: false
+    isGlobal:
+      title: Is Global
+      description: isGlobal can be true or false. There is only one default destination list of type allow or block for an organization.
+      type: boolean
+      required: false
+    name:
+      title: Name
+      description: Name of the DL list
+      type: string
+      required: false
+    thirdpartyCategoryId:
+      title: Third Party Category Id
+      description: example, 0
+      type: integer
+      required: false
+    createdAt:
+      title: Created At
+      description: example, 1490206249
+      type: date
+      required: false
+    modifiedAt:
+      title: Modified At
+      description: example, 1490206249
+      type: date
+      required: false
+    isMspDefault:
+      title: Is MSP Default
+      description: example, false
+      type: boolean
+      required: false
+    markedForDeletion:
+      title: Marked For Deletion
+      description: example, false
+      type: boolean
+      required: false
+    meta:
+      title: Meta Data
+      description: None
+      type: meta
+      required: false
+  dlCollection:
+    id:
+      title: ID
+      description: Unique id of the destination list.
+      type: integer
+      required: false
+    organizationId:
+      title: Organization Id
+      description: ID of org, e.g. 2345678
+      type: integer
+      required: false
+    access:
+      title: Access
+      description: Access can be allow or block. It defines destinationList type, e.g. allow
+      type: string
+      required: false
+    isGlobal:
+      title: Is Global
+      description: isGlobal can be true or false. There is only one default destination list of type allow or block for an organization.
+      type: boolean
+      required: false
+    name:
+      title: Name
+      description: Name of the DL list
+      type: string
+      required: false
+    thirdpartyCategoryId:
+      title: Third Party Category Id
+      description: example, 0
+      type: integer
+      required: false
+    createdAt:
+      title: Created At
+      description: example, 1490206249
```

Comment:
PluginValidator also requires an example field for each input so you could move this example and others like it into a separate example field